src/workspace/block_entry.js

```diff
@@ -17813,12 +17866,12 @@ Entry.block = {
             value = Math.round(value);
             value = Math.max(value, 1);
             value = Math.min(value, 179);
-            Entry.hw.setDigitalPortValue(port, value);
+            Entry.hw.setDigitalPortValue(9, value);
             return script.callReturn();
         }
     },
     "dplay_Robot_run": {
-        "color": "#00979D",
+        "color": "#005766",
```

Comment:
네. 로보쪽 블럭만 색상을 변경했습니다혹시. 무슨 문제되는 부분이 있을까요?

네이버 메일앱에서 보냈습니다.-----Original Message-----
보낸사람: "JY Kim"&lt;notifications@github.com&gt;
받는사람: "entrylabs/entryjs"&lt;entryjs@noreply.github.com&gt;
참조: "SungminSeung"&lt;seungsm@naver.com&gt;, "Author"&lt;author@noreply.github.com&gt;
날짜: 2016. 6. 9 오후 6:28:35
제목: Re: [entrylabs/entryjs] modify block (#341)

In src/workspace/block_entry.js:
&gt;              return script.callReturn();&gt;          }&gt;      },&gt;      "dplay_Robot_run": {&gt; -        "color": "#00979D",&gt; +        "color": "#005766",로봇블록들은 색상이 다르던데 의도된 것인가요?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.